spec/factories/trainees.rb

```diff
@@ -40,9 +40,12 @@
     end
 
     factory :trainee_for_form do
-      add_attribute("date_of_birth(3i)") { Faker::Date.birthday(min_age: 18, max_age: 65).day.to_s }
-      add_attribute("date_of_birth(2i)") { Faker::Date.birthday(min_age: 18, max_age: 65).month.to_s }
-      add_attribute("date_of_birth(1i)") { Faker::Date.birthday(min_age: 18, max_age: 65).year.to_s }
+      transient do
+        date_of_birth { Faker::Date.birthday(min_age: 18, max_age: 65) }
```

Comment:
is this `date_of_birth` the exact same on the model?

Reply:
renamed the var so its more obvious that it does not relate to the model.